packages/frontend/core/src/modules/peek-view/view/peek-view-controls.tsx

```diff
@@ -124,6 +124,15 @@ export const DocPeekViewControls = ({
           peekView.close('none');
         },
       },
+      {
```

Comment:
desktop only?

Reply:
open in new tab is now also available for web